plugins/autoclothing.cpp

```diff
@@ -523,7 +565,7 @@ static void find_needed_clothing_items()
 
                 if (!item)
                 {
-                    WARN(cycle).print("Invalid inventory item ID: %d\n", ownedItem);
+                    WARN(cycle).print("autoclothing: Invalid inventory item ID: %d\n", ownedItem);
```

Comment:
`debugfilter` has the capability to turn on source plugin labels in debug messages, but it's a configurable header element. What if, instead of tagging here (and below), we make that header element always on for WARN and above?

https://github.com/DFHack/dfhack/blob/develop/library/Debug.cpp#L152

Reply:
i have mixed feelings about this idea, it'll force edits to a number of plugins that already tag all their messages to avoid doubletagging

Reply:
we can keep it as a follow-up issue, then. I think plugins tagging their messages manually is redundant, but it doesn't have to be solved in this PR